sql/src/test/resources/drill/window/queries/nestedAggs/multiWin_5.e

```diff
@@ -8,15 +8,15 @@ false	280487665.00748299428571428571
 false	295757331.64717262000000000000
 false	302662813.16785714333333333333
 false	304608131.82107142900000000000
-false	303537640.51502361272727272727
+false	303537640.515023651272727272727
```

Comment:
These queries do not compare the query plan. The underlying query plan changed a bit due to removal of the rule hence the change in the terminating decimal places